### ISSUE TITLE
Question about evaluation on SemEval2018

### ISSUE THREAD

**Issue by zorache** [hasanhuz/SpanEmo#7]
Hi!

I've trained on English SemEval2018 train and dev splits with default configurations, but haven't been able to get similar test results on SemEval2018 test split (E-c-En-test-gold.txt specifically). 

For average of three models with the same default setting, I got 
F1-Micro 0.6953, F1-Macro 0.53, JS 0.567

I read your results from your paper being higher, especially F1-Macro with around 4% difference. I want to reach out and see if there's something else beyond the default setting that I should note to reproduce the results. 

Thank you so much.

**Comment by hasanhuz** [comment]
Hi, thank you for your interest in our work. From your results, it seems to me that you didn't use the training objective introduced in the paper. Please, ensure that both the correlation loss and binary cross-entropy are enabled. The alpha value should be set 0.2 as described in the paper. 
Hope this helps! 
Hassan

**Comment by hasanhuz** [comment]
Hi @zorache, since you haven't posted here for a while, I assume that your problem was resolved. Please, free to open a new issue if you still face the same problem!
Best wishes,
